app/views/layouts/_error_tracking.html.erb

```diff
@@ -1,5 +1,5 @@
 
-  <% if Rails.env.production? %>
+  <% if false %>
```

Comment:
Disabling this for now... it isn't working anyway. Error reporting will get an overhaul once we're done with assets.